### DISCUSSION TITLE
`sentry-android-fragment` documentation

### DISCUSSION THREAD

**Discussion by mateuszkwiecinski** [getsentry/sentry-java#1735] [Q&A] [answered]
👋 
I wanted to ask if there is any documentation on how to use `sentry-android-fragment` integration. When I wanted to try it out I found difficult to 
a) find anything than release notes mentioning it exists
b) find how to integrate new integration with existing sentry config (I went through checking classes available in new artifact, looking at interface it implements, and guessing how to include an `Integeration` to Sentry setup
c) It is also unclear what `enableFragmentLifecycleBreadcrumbs` and `enableAutoFragmentLifecycleTracing` do. There are no kDocs over these properties, docs page doesn't produce any results:
![image](https://user-images.githubusercontent.com/36954793/134321775-c3fda8e6-7945-4ecf-9c9f-c40b4be9359b.png)

**Comment by marandaneto** [comment, marked as answer]
@mateuszkwiecinski does that help? https://docs.sentry.io/platforms/android/configuration/integrations/fragment/ and also https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-fragment-instrumentation

**Reply by mateuszkwiecinski** [reply]
It does. Thank you! Can you help me find the way I should have located it on my own? Was it linked somewhere? 

I'm having similar problem for Apollo integration added in `5.2.0`, I can't find it on on sentry docs, near other integrations
![image](https://user-images.githubusercontent.com/36954793/134323808-8cfa4366-bb62-41b3-883d-40d3938ee96b.png)

Search doesn't produce valid results here as well:
![image](https://user-images.githubusercontent.com/36954793/134324084-1b134b5d-2c1b-4ab2-82ec-2a5acb6063b1.png)

**Reply by marandaneto** [reply]
apollo isnt documented yet, its in the making -> https://github.com/getsentry/sentry-docs/issues/4151
